torch_xla/csrc/runtime/distributed_runtime.h

```diff
@@ -0,0 +1,35 @@
+#ifndef XLA_CLIENT_DISTRIBUTED_RUNTIME_H_
+#define XLA_CLIENT_DISTRIBUTED_RUNTIME_H_
+
+#include <memory>
+
+#include "xla/pjrt/distributed/distributed.h"
+
+namespace torch_xla {
+namespace runtime {
+
+class DistributedRuntime {
+ public:
+  static DistributedRuntime& getInstance(int global_rank) {
+    static DistributedRuntime dist_runtime_instance(global_rank);
+    return dist_runtime_instance;
+  }
+  ~DistributedRuntime();
+  DistributedRuntime(DistributedRuntime const&) = delete;
+  void operator=(DistributedRuntime const&) = delete;
+
+  std::shared_ptr<xla::DistributedRuntimeClient> GetClient(int global_rank);
```

Comment:
If we move to `Initialize`, we won't need the global_rank as a parameter here.

Reply:
Actually, if you see distributed_runtime.cc, this `global_rank` is not needed in the function so can be removed.